README.md

```diff
@@ -1,70 +1,78 @@
-# Getting Started with Create React App
+![](https://img.shields.io/badge/Microverse-blueviolet)
 
-This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
+# Space Traveler Hub
 
-## Available Scripts
+> Build a page that shows missions and rockets feched from the SpaceX api and allows the user to reserve both missions and rockets.
 
-In the project directory, you can run:
+<div align="center"><img src="./spaceX.png" width="500"/></div>
 
-### `npm start`
+![image](https://user-images.githubusercontent.com/81584449/132874276-9bab7d78-ad21-443d-9cf0-f4ccc628d929.png)
 
-Runs the app in the development mode.\
-Open [http://localhost:3000](http://localhost:3000) to view it in the browser.
+Mobile ScreenShots
 
-The page will reload if you make edits.\
-You will also see any lint errors in the console.
+## Built With
 
-### `npm test`
+- React.
+- Redux.
+- JavaScript.
+- SpaceX api
 
-Launches the test runner in the interactive watch mode.\
-See the section about [running tests](https://facebook.github.io/create-react-app/docs/running-tests) for more information.
+## Video
 
-### `npm run build`
+[Link](https://www.loom.com/share/a451a571209c414b8663b7d8a2a812ba) to video on Loom.
```

Comment:
- [ ] Good job presenting your capstone 👏 However, I believe that @eri8-9 voice doesn't seem to be obvious at all! 🙂 So, I'm afraid you'll have to record another presentation video 🙏